cmd/create.go

```diff
@@ -194,6 +198,7 @@ func init() {
 	componentCreateCmd.Flags().StringVarP(&componentGit, "git", "g", "", "Use a git repository as the source file for the component")
 	componentCreateCmd.Flags().StringVarP(&componentLocal, "local", "l", "", "Use local directory as a source file for the component")
 	componentCreateCmd.Flags().StringSliceVar(&componentPorts, "port", []string{}, "Ports to be used when the component is created")
+	componentCreateCmd.Flags().StringSliceVar(&componentEnvVars, "env", []string{}, "Environmental variables for the component, specified like --env VariableName=Value")
```

Comment:
small nitpick. don't use "like". Use `For example`.

So in this case:

`Environment variables for the component. For example --env Key=Value`

We also shouldn't use "VariableName", we should keep consistent with using the whole `Key=Value` examples.

Reply:
But @kadel suggested me to use VariableName instead of key https://github.com/redhat-developer/odo/pull/626#discussion_r220176905

Reply:
Ah you're right. Let's do that.